core/chains/terra/terratxm/txm.go

```diff
@@ -269,21 +239,33 @@ func (txm *Txm) sendMsgBatchFromAddress(gasPrice sdk.DecCoin, sender sdk.AccAddr
 		return
 	}
 
-	if err := txm.confirmTx(resp.TxResponse.TxHash, getSimMsgsIDs(simResults.Succeeded)); err != nil {
+	if err := txm.confirmTx(resp.TxResponse.TxHash, simResults.Succeeded.GetSimMsgsIDs()); err != nil {
 		txm.lggr.Errorw("error confirming tx", "err", err, "hash", resp.TxResponse.TxHash)
 		return
 	}
 }
 
+func (txm *Txm) confirmPollConfig() (maxPolls int, pollPeriod time.Duration) {
+	blocks := txm.cfg.BlocksUntilTxTimeout()
+	blockPeriod := terra.BlockRateSeconds * time.Second
+	pollPeriod = txm.cfg.ConfirmPollPeriod()
+	if pollPeriod == 0 { // don't divide by zero
+		return 2, 100 * time.Millisecond
```

Comment:
@connorwstein This base case was chosen to keep `TestTxm/failed_to_confirm` passing, which sets poll period to zero. Perhaps something else like `return 1, 0` makes more sense, but it wasn't immediately clear to me how to fix the test.

Reply:
we could make the poll period and max poll arguments to confirmTx and then pass in the test values in the test